tutorials/lb-firewall-haproxy-pfsense/index.mdx

```diff
@@ -1,9 +1,9 @@
 ---
 meta:
-  title: How to Build a Redundant Loadbalancer and Firewall Infrastructure With pfSense and HAproxy on a Scaleway Dedibox
+  title: Building a Redundant Loadbalancer and Firewall Infrastructure With pfSense and HAproxy on a Scaleway Dedibox
   description: This page explains how to install pfSense on a Scaleway Dedibox dedicated server
 content:
-  h1: How to Build a Redundant Loadbalancer and Firewall Infrastructure With pfSense and HAproxy on a Scaleway Dedibox
+  h1: Building a Redundant Loadbalancer and Firewall Infrastructure With pfSense and HAproxy on a Scaleway Dedibox
```

Comment:
Building a redundant load balancer and firewall infrastructure with pfSense and HAproxy on a Scaleway Dedibox